libmproxy/proxy/config.py

```diff
@@ -133,10 +133,12 @@ def process_proxy_options(parser, options):
 
     if options.clientcerts:
         options.clientcerts = os.path.expanduser(options.clientcerts)
-        if not os.path.exists(options.clientcerts) or not os.path.isdir(options.clientcerts):
+        if not (os.path.exists(options.clientcerts) or
```

Comment:
I was under the impression that using these function calls would help deter from special files being used (device files, fifo, etc.), in which case os.path.exists() could be removed. But since I could not find any mention of special files on https://docs.python.org/2/library/os.path.html I am not sure Python has such support. Thus, I'll go ahead and follow your recommendation and remove the isFile and isDir calls.